modules/mobile/spec/request/appointments_vaos_v2_list_request_spec.rb

```diff
@@ -184,7 +184,13 @@
         expect(response.body).to match_json_schema('VAOS_v2_appointments')
 
         uniq_statuses = response.parsed_body['data'].map { |appt| appt.dig('attributes', 'status') }.uniq
-        expect(uniq_statuses).to eq(%w[CANCELLED BOOKED])
+        expect(uniq_statuses).to eq(%w[BOOKED CANCELLED])
```

Comment:
Maybe we should be using `.match_array` (which is order agnostic) instead of `.eq` to avoid this kind of problem. I'm probably the person who started this test pattern, so probably my fault.

Reply:
good idea. fixed